### ISSUE TITLE
XCFramework is not properly generated for arvm7 architecture

### ISSUE THREAD

**Issue by luisrecuenco** [shakebugs/shake-ios#13]
It seems that from version 14.1.0, the XCFramework no longer has the correct arvm7 slice and it fails to compile with the following message.

```
Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_SHKShake", referenced from:
      objc-class-ref in AppDelegate.o
ld: symbol(s) not found for architecture armv7
```

Version 12.1.0 is properly generated though.

**Comment by luisrecuenco** [comment]
I think the problem is that the `Package.swift` is setting the minimum version to iOS 10 when it should be iOS 11, based on the documentation. iOS 10 devices still require the armv7 architecture for 32 bits, unlike iOS 11.

**Comment by david-decode** [comment]
Hi @luisrecuenco ,

Good catch, we bumped the minimum version to iOS11, but looks like we forgot to edit the Package.swift file.
We'll test it out and the next release should contain the fix for this.

Thanks for the detailed report.

David

Shake Team

**Comment by david-decode** [comment]
Fixed with the 14.1.3 release
